packages/common/styles/_tokens.scss

```diff
@@ -1,8 +1,23 @@
 @import './colors';
 
-// Font tokens
+// Font-face and weights
+// Weights:
```

Comment:
@moneppo let me know if you have thoughts on these weights and the naming conventions here. we have "Gotham Light" in our fonts library, but we already determined that it's never used (we don't even define it as a font-face in Code Studio). i left room to add it here, but left it out for now

Reply:
also, side note: i tried to stick to the `font-weight` CSS definition as much as possible here: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#syntax